src/cow-react/modules/limitOrders/hooks/useLimitOrdersTradeState.ts

```diff
@@ -20,6 +20,24 @@ export interface LimitOrdersTradeState {
   readonly outputCurrencyFiatAmount: CurrencyAmount<Currency> | null
   readonly recipient: string | null
   readonly deadline: number | null
+  readonly isWrap: boolean
+  readonly isUnwrap: boolean
+}
+
+function detectWrapUnwrap(
```

Comment:
Very nice, I would move this to some shared/global space as util function because it seems like we might want to re-use this in for other purposes.

Reply:
Also just to mention there is this util function https://github.com/cowprotocol/cowswap/blob/ea734d030bb2e9392da2bcfb8d9cb2792f3f25f5/src/custom/state/swap/utils.ts#L6